presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveStorageFormats.java

```diff
@@ -200,6 +201,19 @@ public void testCreatePartitionedTableAs(StorageFormat storageFormat)
         query(format("DROP TABLE %s", tableName));
     }
 
+    @Test(groups = STORAGE_FORMATS)
+    public void testOrcTableCreatedInPresto()
+    {
+        onPresto().executeQuery("CREATE TABLE orc_table_created_in_presto WITH (format='ORC') AS SELECT 42 a");
+        assertThat(onHive().executeQuery("SELECT * FROM orc_table_created_in_presto"))
+                .containsOnly(row(42));
+        assertThat(onHive().executeQuery("SELECT * FROM orc_table_created_in_presto"))
+                .containsOnly(row(42));
```

Comment:
Can we add a comment why there're duplicated queries here? I understood it is required after seeing https://github.com/prestosql/presto/issues/456#issuecomment-538325618 though.

Reply:
Right, added a comment.